pinot-core/src/test/java/org/apache/pinot/core/operator/transform/function/CastTransformFunctionTest.java

```diff
@@ -40,21 +40,21 @@ public void testCastTransformFunction() {
     testTransformFunction(transformFunction, expectedValues);
 
     expression = RequestContextUtils.getExpressionFromSQL(
-        String.format("CAST(ADD(CAST(%s AS INT), %s) AS STRING)", STRING_SV_COLUMN, DOUBLE_SV_COLUMN));
+        String.format("CAST(ADD(CAST(%s AS LONG), %s) AS STRING)", DOUBLE_SV_COLUMN, LONG_SV_COLUMN));
```

Comment:
can you add more test coverage? There are quite a few special handling logic added.